app/_src/networking/transparent-proxy/universal.md

```diff
@@ -0,0 +1,468 @@
+{% endwarning %}
+
+<!-- vale Google.Headings = NO -->
+### YAML / JSON
+<!-- vale Google.Headings = YES -->
+
+You can provide the configuration in either `YAML` or `JSON` format by using the `--config` or `--config-file` flags.
+
+{% tip %}
+For the configuration schema in YAML format, refer to the [Schema]({{ docs }}/reference/transparent-proxy-configuration/#schema) section in the [Transparent Proxy Configuration Reference]({{ docs }}/reference/transparent-proxy-configuration/).
+{% endtip %}
+
+{% tip %}
+For simplicity, the following examples use YAML format, but you can easily convert them to JSON if preferred. Both formats work exactly the same, so feel free to choose the one that best suits your needs.
+{% endtip %}
+
+Below are examples of using these flags in different ways:
+
+1. **Providing configuration via the `--config-file` flag**
+
+   Assume you have a `config.yaml` file with the following content:
+
+   ```yaml
+   kumaDPUser: dataplane
+   verbose: true
+   ```
+
+   You can install the transparent proxy using:
+
+   ```sh
+   kumactl install transparent-proxy --config-file config.yaml
+   ```
+
+2. **Passing configuration directly via the `--config` flag**
+
+   To pass the configuration content directly:
+
+   ```sh
+   kumactl install transparent-proxy --config "kumaDPUser: dataplane\nverbose: true"
+   ```
+
+   Alternatively:
+
+   ```sh
+   kumactl install transparent-proxy --config "{ kumaDPUser: dataplane, verbose: true }"
+   ```
+
+   Both formats are valid YAML inputs.
+
+3. **Passing configuration via [STDIN](https://en.wikipedia.org/wiki/Standard_streams#Standard_input_(stdin))**
+
+   If you need to pass the configuration via STDIN, set `--config-file` to `-` as shown below:
+
+   ```sh
+   echo "
+   kumaDPUser: dataplane
+   verbose: true
+   " | kumactl install transparent-proxy --config-file -
+   ```
+
+<!-- vale Google.Headings = NO -->
+### Environment Variables
+<!-- vale Google.Headings = YES -->
+
+You can customize configuration settings by using environment variables. For example:
+
+```sh
+KUMA_TRANSPARENT_PROXY_IP_FAMILY_MODE="ipv4" kumactl install transparent-proxy
+```
+
+{% tip %}
+To see all available environment variables, visit the [Environment Variables]({{ docs }}/reference/transparent-proxy-configuration/#environment-variables) section in the [Transparent Proxy Configuration Reference]({{ docs }}/reference/transparent-proxy-configuration/).
+{% endtip %}
+
+<!-- vale Google.Headings = NO -->
+### CLI Flags
+<!-- vale Google.Headings = YES -->
+
+Most configuration values can also be specified directly through CLI flags. For example:
+
+```sh
+kumactl install transparent-proxy --kuma-dp-user dataplane --verbose
+```
+
+{% warning %}
+The following settings cannot be modified directly via CLI flags (corresponding flags are not available):
```

Comment:
I think we discussed not offering CLI flags in the future. Should we just stop documenting this now?